### PR TITLE
Dependencies

### LINKED ISSUE #8: Views cannot depend on other views
You cannot define a view which uses another view and reliably get the `sync_pgviews` command to work correctly.

Proposed solution:

```python
class MyView(View):
    class Meta:
         dependencies = ('my_app.View', 'my_app.Otherview',)
```

### PR DESCRIPTION
**Pull request by kennydude**
- Rework some of the syncing to support dependencies
- Limit post_migrate signal handler until the final signal
- Implement dependencies

Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +51,8 @@ class Customer(models.Model):
         app_label = 'myapp'
 
 class PreferredCustomer(pg.View):
-    projection = ['myapp.Customer.*']
+    projection = ['myapp.Customer.*',]
+    dependencies = ['myapp.Customer',]
```
Comment (scott-w) [inline README.md]: Would it be possible to get this into the `class Meta` please? Long-term, I'd like to have the meta-properties there.

Comment (kennydude) [inline README.md]: I would have loved to, but Django says no :(

File: django_pgviews/models.py [hunk 2]
```diff
@@ -1,26 +1,89 @@
...
-            msg = "already exists, skipping"
-        elif status == 'FORCED':
-            msg = "forced overwrite of existing schema"
-
-        log.info("%(python_name)s (%(view_name)s): %(msg)s" % {
-            'python_name': python_name,
-            'view_name': view_cls._meta.db_table,
-            'msg': msg})
+    global counter
+    counter = counter + 1
```
Comment (scott-w) [inline django_pgviews/models.py]: Is it possible to have the caller for `sync_pgviews` manage this counter for us?

Comment (kennydude) [inline django_pgviews/models.py]: Yeah it does look really odd, i'll see what I can do

Comment (kennydude) [inline django_pgviews/models.py]: The sender is the application which finished migrating, so sadly this is not something we can do :(

Comment (scott-w) [inline django_pgviews/models.py]: However, we use a custom `__metaclass__` here that could `pop` the additional attributes before the parent `__new__` is called, so maybe that could work?

File: README.md (hunk 1 above)
Comment (scott-w) [inline README.md]: However, we use a custom `__metaclass__` here that could `pop` the additional attributes before the parent `__new__` is called, so maybe that could work?

Comment (kennydude) [inline README.md]: I think I did try it and it just doesn't work :/

File: django_pgviews/models.py (hunk 2 above)
Comment (scott-w) [inline django_pgviews/models.py]: Could you at least use a decorator that documents what it's doing? I'm just not a fan of modifying a global variable that any other part of the system could access.

Comment (kennydude) [inline django_pgviews/models.py]: Fixed it by moving it somewhere else

**Pull request merged**